README.md

```diff
@@ -79,6 +79,10 @@ in the Kubernetes documentation.
 
 Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 
-## Slack Channels
+## Contact
 
-Join us at [Rook ceph-csi Channel](https://rook-io.slack.com/messages/CG3HUV94J/details/)
+Please use the following to reach members of the community:
+
+- Slack: Join our [slack channel](https://cephcsi.slack.com)
```

Comment:
(query) We are going to have a new slack channel? Or, is this the same as the older one under Rook?

Reply:
same question. need to know the reason behind creating new channel :)

Reply:
Actually its different. Lets hear the feedback.

Reply:
@Madhu-1 @ShyamsundarR Its not necessary that, all CSI users are rook users. Atleast I came to know recently that there are enough users who use ceph-csi seperately. Thats the justification. :)

Reply:
we have around 86 users in current csi slack channel. what about them?

Reply:
We have 86 members in the other (rook managed) slack channel, are we going to maintain 2? Or, are we going to announce it to the channel that we are moving, or what is the plan here?

From whom are we expecting to hear feedback on for the new channel?

Reply:
>From whom are we expecting to hear feedback on for the new channel?
@ShyamsundarR people who have contributed here or close to us, the list is avialable as a comment below. So that we can conclude based on majority  :) 
Thanks ..

Reply:
It does seem to me that having a more "general" forward facing channel for CSI isn't a bad idea.  Pointing to the Rook channel is ok too IMO (at least for now).  I do think it might be worth looking at a general "csi-users" slack channel under the Kubernetes slack that we just "share".  A lot of questions that come in tend to be general CSI questions so it would make sense to me.

Reply:
@j-griffith thanks for sharing your thought! yeah, having a seperate channel to accommodate every ceph-csi users/dev..etc is better for long term. Regarding sharing kubernetes slack, it is a good idea, but unfortunately I dont think its going to happen. afaict, unless and until we/ceph-csi are hosted under kube, its  not possible.